src/sys/DATA.ASM

```diff
@@ -886,7 +882,13 @@ get_item_list:                                  ; Get a specific item from a com
 
 .quotation:
     xor bl, 0xFF                                ; Toggle comma-escape
-    jmp .find_comma                             ; Immediately read the next byte
+
+    cmp byte [si], 0x22                         ; Is the next byte another double-quote?
+    jne .find_comma                             ; If not, proceed
+
+    xor bl, 0xFF                                ; Otherwise, reset comma-escape
+    inc si                                      ; Increment source index
+    jmp .write                                  ; And write one double-quote if necessary
```

Comment:
Comma-separated lists use a pair of double-quotes to denote a single double-quote to avoid confusion with the string delimiter.